Tests/Test_Voltage.c

```diff
@@ -84,3 +188,36 @@ int main() {
 
     while(1);
 }
+
+// Task for this test
+void Task1(void *p_arg){
+	OS_CPU_SysTickInit(SystemCoreClock / (CPU_INT32U) OSCfg_TickRate_Hz);
+	
+    test();
+}
```

Comment:
Do we need to put a blank while loop here? I'm not sure how this will act without an idle task to fall back on

Reply:
Added this just now to the end of `test()`